ui/explore/src/components/PrometheusMetricsFinder/overview/MetricOverview.tsx

```diff
@@ -0,0 +1,202 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Skeleton, Stack, StackProps, Tab, Tabs, Tooltip } from '@mui/material';
+import { useMemo, useState } from 'react';
+import { DatasourceSelector, Definition, QueryDefinition, UnknownSpec } from '@perses-dev/core';
+import { Panel } from '@perses-dev/dashboards';
+import useResizeObserver from 'use-resize-observer';
+import { DataQueriesProvider, useSuggestedStepMs } from '@perses-dev/plugin-system';
+import HelpCircleOutlineIcon from 'mdi-material-ui/HelpCircleOutline';
+import { computeFilterExpr, LabelFilter } from '../types';
+import { useMetricMetadata } from '../utils';
+import { OverviewTab } from './tabs/OverviewTab';
+import { JobTab } from './tabs/JobTab';
+import { SimilarTab } from './tabs/SimilarTab';
+
+export interface OverviewPanelProps extends StackProps {
+  metricName: string;
+  datasource: DatasourceSelector;
+  filters: LabelFilter[];
+  type?: string;
+  isLoading?: boolean;
+}
+
+export function OverviewPanel({ metricName, datasource, filters, type, isLoading, ...props }: OverviewPanelProps) {
+  const { width, ref: panelRef } = useResizeObserver();
+  const suggestedStepMs = useSuggestedStepMs(width);
+
+  const { queries, definitions }: { queries: QueryDefinition[]; definitions: Array<Definition<UnknownSpec>> } =
+    useMemo(() => {
+      const expr =
```

Comment:
Would be good to apply a rate() in the case of histograms & summaries also, when applicable:  even if most of the time they produce counter-like timeseries, it can also not be the case, ref https://prometheus.io/docs/practices/histograms/#count-and-sum-of-observations
Maybe a checkbox to enable/disable rate for these?

Reply:
Added a checkbox